src/framework/uicomponents/qml/MuseScore/UiComponents/PageTabButton.qml

```diff
@@ -131,10 +131,16 @@ RadioDelegate {
         implicitWidth: root.leftPadding + contentRow.implicitWidth + root.rightPadding
         implicitHeight: contentRow.implicitHeight
 
+        FontMetrics {
+            id: fontMetricsSelected
+            font: root.selectedStateFont
+        }
+
         Row {
             id: contentRow
             anchors.fill: parent
             spacing: root.spacing
+            width: textLabel.width
```

Comment:
This change seems risky in combination with `anchors.fill`, plus that it seems undesired when there is also an icon besides the text. 
Instead, better set the `width` property of `textLabel`.